inc/audiences/namespace.php

```diff
@@ -755,3 +764,49 @@ function build_audience_query( array $audience ) : array {
 
 	return $groups_query;
 }
+
+/**
+ * Output audience configuration data as JSON.
+ *
+ * @return array
+ */
+function get_audience_config() : array {
+	// Note this cache is cleared in save_audience().
+	$config = wp_cache_get( 'audiences', 'altis.analytics' );
+
+	if ( $config ) {
+		return $config;
+	}
+
+	/**
+	 * Limits the number of active audiences there can be in use at any one time
+	 * on the front end.
+	 *
+	 * @param int $limit The number of audiences to fetch for use client side.
+	 */
+	$limit = apply_filters( 'altis.analytics.audiences.limit', 20 );
+	$limit = absint( $limit );
```

Comment:
should this be `intval`, or do you specifically want to disallow `-1`?

Reply:
Yeah `-1` means 'all' in this context which we don't want